lib/rails_semantic_logger/active_record/log_subscriber.rb

```diff
@@ -130,7 +130,7 @@ def bind_values_v6_1(payload)
       def render_bind_v4_2(column, value)
         if column
           if column.binary?
-            # This specifically deals with the PG adapter that casts bytea columns into a Hash.
+            # This specifically deals with the PG adapter that casts byte a columns into a Hash.
```

Comment:
This seems to be a reference to the [`bytea` Postgres type](https://www.postgresql.org/docs/current/datatype-binary.html), I'm not sure it's a typo? 🤔

Reply:
Ah... I see. Indeed it is. This PR is closed.